test/controllers/merchants_controller_test.rb

```diff
@@ -1,34 +1,37 @@
 require "test_helper"
 
 describe MerchantsController do
-  # it "should get new" do
-  #   get merchants_new_url
-  #   value(response).must_be :success?
-  # end
-  #
-  # it "should get create" do
-  #   get merchants_create_url
-  #   value(response).must_be :success?
-  # end
-  #
-  # it "should get index" do
-  #   get merchants_index_url
-  #   value(response).must_be :success?
-  # end
-  #
-  # it "should get edit" do
-  #   get merchants_edit_url
-  #   value(response).must_be :success?
-  # end
-  #
-  # it "should get update" do
-  #   get merchants_update_url
-  #   value(response).must_be :success?
-  # end
-  #
-  # it "should get show" do
-  #   get merchants_show_url
-  #   value(response).must_be :success?
-  # end
+  let(:merchant) {merchants(:sappy1)}
+  let(:other_merchant) {merchants(:sappy2)}
+
+  describe "logged in merchants" do
+
+    it "merchants can access their own account page" do
+        login(merchant, :github)
```

Comment:
probably want to test that the merchant is logged in (either a flash message check or the session[:merchant_id])